### DISCUSSION TITLE
Odd behavior in the ice field

### DISCUSSION THREAD

**Discussion by GeorgeGayno-NOAA** [ufs-community/ufs-weather-model#1954] [Q&A] [answered]
I am adding a new regression test based on control_p8. This new test is the same as control_p8, except it uses coldstart files from a new fractional grid version of chgres_cube. 

I am getting an odd result in the ice fields. I have attached gifs of the ice concentration and ice temperature (TISFC) from the coldstart file (tile6, left) and from the 00-hour tiled history file (right).

Note that the ice concentration changes between the coldstart and 00-hour file. Why is that?

And since chgres_cube only outputs a TISFC at points that it thinks is ice (the white area is a missing flag), the 00-hour file contains a large area of very cold TISFC (the blue area, which is 173K) at these "new" ice points. Presumably, this very cold area is the result of some kind of range check.

Whom should I ask for assistance?

**Comment by GeorgeGayno-NOAA** [comment]
Ice concentration, tile 6. Left is the coldstart file. Right is the 00-hr file.

![Ice_concentration](https://github.com/ufs-community/ufs-weather-model/assets/52789452/2da7b9d7-3ae3-4b18-bc68-6263d8f934b3)

**Reply by GeorgeGayno-NOAA** [reply]
Note how the ice area expands.

**Comment by GeorgeGayno-NOAA** [comment]
Ice temperature (TISFC). Left, the coldstart file. Right, the 00-hr file. Tile 6. Large blue area in the 00-hr file is 173K.


![ice temperature](https://github.com/ufs-community/ufs-weather-model/assets/52789452/1c69e593-923e-4373-9872-38b6916df1e3)

**Reply by GeorgeGayno-NOAA** [reply]
The white area is a flag value of -1E+20.

**Comment by yangfanglin** [comment]
@GeorgeGayno-NOAA Is control_p8 coupled or uncoupled ?

**Reply by GeorgeGayno-NOAA** [reply]
@junwang-noaa says that test is ATM only.

**Comment by DeniseWorthen** [comment]
The F00 hour forecast file is really after one timestep, correct? For example, the sfc000 file for the coupled model is at 0.2 (720s) after the initialization time. I don't know that explains what you're seeing though.

**Comment by yangfanglin** [comment]
What is the water temperature at those "blue" points in the coldstart ICs ?   New sea-ice can form over cold water below 271K or so.  Then it is a question how TISFC and/or TMPSFC are assigned/determined.  CICE might be doing this differently than the uncoupled thermodynamic ice model.   @NeilBarton-NOAA  @Xingren.Wu@noaa.gov

**Reply by GeorgeGayno-NOAA** [reply]
Here is the TSEA (SST) record from the coldstart file. It (should) have a value for any point with at least some open water.
 
![TSEA_coldstart](https://github.com/ufs-community/ufs-weather-model/assets/52789452/595d1637-7647-49a2-9076-b1b203439878)

The SST in the area of the odd TISFC is around 271.3K

**Reply by yangfanglin** [reply]
we need to check the code to see how ice surface temperature is determined for newly formed sea ice.

**Comment by yangfanglin** [comment]
Comments from Xingren:
--------------------
I quickly went through the link you provided. It seems that this is an ATM only run. In the GFS we usually have sea ice fixed w/out change, sea ice can melt but does not form. It is possible this is due to sea ice analysis/climatology update from "sfccycle" but we need to check the code and "namelist" settings for the test case.

Anyway, when new sea ice forms in the GFS (such as in the AMIP run, or following sea ice analysis update, or in the coupled run), sea ice temperature (TISFC) over new ice is set at the freezing point of seawater (fixed at 271.2K, I think; but in the coupled case, such as CFSv2, or UFS, TISFC should come from the coupler).

I would suggest checking the GFS opr run, or a run similar, and do the same plots.

---------------------------

**Comment by GeorgeGayno-NOAA** [comment]
Upon @junwang-noaa's suggestion, I checked the sfcsub.F nudging coefficients for ice - FSICL and FSICS. They are set to '0', which means the ice climatology is applied as the model runs. I changed both to '99999'. The ice no longer changes at the 00-hour point. Will do some more tests.

**Comment by GeorgeGayno-NOAA** [comment]
Adjusting the nudging coefficients for ice, appears to correct the problem.

On the left is the ice concentration from the coldstart file. On the right is the ice concentration at the 00-hour point. They look the same.
![ice_concentration](https://github.com/ufs-community/ufs-weather-model/assets/52789452/1d00314c-761d-4331-955e-1849ddf51b82)

On the left/right is the ice temperature before/after the change to the coefficient. The area of very cold temperature is gone.

![ice_temperature](https://github.com/ufs-community/ufs-weather-model/assets/52789452/6d48cfc1-edf3-4e23-a63b-33155d6b8cee)

And on the left/right, the ice depth before/after the change. The large ice depths are gone.

![ice_depth](https://github.com/ufs-community/ufs-weather-model/assets/52789452/d3c26ad6-004e-4def-be71-0cca5b78e30d)

**Comment by GeorgeGayno-NOAA** [comment]
Xingren.Wu@noaa.gov my test directory is here. Can you check? 

`/scratch1/NCEPDEV/stmp2/George.Gayno/FV3_RT/rt_169082.nudging/control_p8.v2.sfc_intel`

**Comment by yangfanglin** [comment]
@GeorgeGayno-NOAA @junwang-noaa  

Was the issue caused by the inconsistency between the following two lines in sfcsub.F for ice thickness and ice concentration ?

      data fsihl/99999.0/,  fsihs/99999.0/
      data fsicl/0.0/,      fsics/0.0/

**Reply by GeorgeGayno-NOAA** [reply]
I don't know. The sea ice climatology does not have a thickness record. So, I don't know if changing that coefficient will actually do anything. 

What is the 'p8' configuration and why does it require climatological ice?

**Comment by GeorgeGayno-NOAA** [comment]
Following the suggestion of Xingren.Wu@noaa.gov, I modified chgres to place a value of 271.2K at points with no ice in the TISFC and TIICE records (however, points that are 100% still retained the large missing flag). So, when ice forms, a reasonable temperature will be available for the ice model. Xingren looked at the results and gave his approval.

**Comment by GeorgeGayno-NOAA** [comment]
One (hopefully) final question. The new chgres places a valid 4-layer soil temperature in the STC array for points that contain at least some land. At other points, a large flag value is used. Looking at the 12-hour restart file for the `control_p8` experiment shows a value of 200K at ice points (blue area). See graphic on the left (layer 1 STC). This value does not change in time, so I assume STC is not used at ice. The same experiment that uses the original coldstart files (which contain an STC value everywhere) is the graphic on the right. It has no very cold area. My question, is there anything wrong with how the new chgres is setting STC? Is STC active at non-land points? 



![STC_lyr_1](https://github.com/ufs-community/ufs-weather-model/assets/52789452/22ce686a-96cc-428b-9282-193a07ed6041)

**Comment by gspetro-NOAA** [comment]
@GeorgeGayno-NOAA Just wanted to check whether this discussion is resolved (and can be closed) or whether further assistance is required. Thanks! :)

**Reply by GeorgeGayno-NOAA** [reply, marked as answer]
Go ahead and close. These changes were merged under #2005.

**Reply by gspetro-NOAA** [reply]
Thanks! :)
